ui/js/controllers/logviewer.js

```diff
@@ -164,7 +164,7 @@ logViewerApp.controller('LogviewerCtrl', [
                 $scope.logViewerTitle = job.get_title();
 
                 if (job.logs && job.logs.length) {
-                    $scope.reftestUrl = `${getReftestUrl(job.logs[0].url)}&only_show_unexpected=1`;
```

Comment:
Is there an actual fix for something (displaying the link to the reftest analyzer?) hiding behind this bug?